code/datums/browser.dm

```diff
@@ -153,31 +153,41 @@
 	opentime = 0
 	close()
 
-//designed as a drop in replacement for alert(); functions the same. (outside of needing User specified)
-/proc/tgalert(mob/User, Message, Title, Button1="Ok", Button2, Button3, StealFocus = 1, Timeout = 6000)
+/**
+ * **DEPRECATED: USE tgui_alert(...) INSTEAD**
```

Comment:
What still uses this that its deprecated and not removed/reimplemented?

Reply:
@bobbahbrown have you considered using a native byond alert for reporting bugs?

Reply:
@Jared-Fogle remember when you told me in discord to not use this for the report issue button but to keep it as legacy tgalert

Reply:
@stylemistake the tgalert for when there are testmerges present on the report bugs alert includes buttons and stuff for the testmerges so it doesn't really lend itself to native alerts